modules/installation-ibm-cloud-export-variables.adoc

```diff
@@ -56,3 +79,15 @@ endif::[]
 ifeval::["{context}" == "installing-ibm-cloud-private"]
 :!ibm-vpc:
 endif::[]
+ifeval::["{context}" == "installing-ibm-power-vs-customizations"]
+:!ibm-power-vs:
+endif::[]
+ifeval::["{context}" == "installing-ibm-power-vs-private-cluster"]
+!:ibm-power-vs:
```

Comment:
s/!:ibm-power-vs:/:!ibm-power-vs:

The Netlify preview shows a stray `!:ibm-power-vs:` tag for the `installing-ibm-power-vs-private-cluster` context.

Reply:
@mburke5678 the private cluster assembly is using the module. Thus added the `ifeval` conditionals to the assembly.
Link : https://47558--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installing-ibm-power-vs-private-cluster.html#installation-ibm-cloud-export-variables_installing-ibm-power-vs-private-cluster

Reply:
@mburke5678 my apologies. I have corrected the module as explained above. Thanks.